### ISSUE TITLE
DESIGN - Figma Kit - Housekeeping - Documentation

### ISSUE THREAD

**Issue by khanhcao17** [hackforla/design-systems#472]
### Overview

Work with content design to research and create a page guiding designers on best practice for creating housekeeping recommendations about how to document their process for their Figma files.

### Action Items

**Creating a quick guide on "documentation Housekeeping recommendations" for other designers.**

- Research
- Working with Content
- Design
     - [ ] [Insert updated date + comment date, a screenshot, and any changes]

### Resources/Instructions

[Figma file](https://www.figma.com/file/5NacfOm45QlrWgr6wWEg4k/Design-System-Template-for-HfLA?node-id=2%3A54&t=r0qPxeKsx29UQhk6-0)
[HfLA Figma Kit](https://www.figma.com/file/XIy3Dciciej4Q6A1TXGMS1/HfLA-Figma-Kit?node-id=2%3A16&t=SAwrdulQxK1A00A7-0)

**Comment by allthatyazz** [comment]
@liz-zheng Please review #470. I added context information such as links to our previous research.

**Comment by khanhcao17** [comment]
### 1/26/2023

![image](https://user-images.githubusercontent.com/83927869/214973849-baa31cfa-56e0-46d6-8579-afd0cb2bb227.png)

**Comment by khanhcao17** [comment]
### 2/2/2023

@liz-zheng Can you add any comments and thoughts about your thought process in creating this page here?

![image](https://user-images.githubusercontent.com/83927869/218222591-63d73101-5fd8-42b4-a525-74ebba7af7e5.png)

**Comment by khanhcao17** [comment]
### 2/2/2023

@liz-zheng Can you add your link to your research here?

**Comment by khanhcao17** [comment]
### 2/2/2023

@liz-zheng Can you add any comments and thoughts about what changes you've made for this newest iteration?

![image](https://user-images.githubusercontent.com/83927869/218222706-f7dfc5c2-149c-4ed2-8603-3a5430a47e20.png)

**Comment by liz-zheng** [comment]
> ### 2/2/2023
> @liz-zheng Can you add any comments and thoughts about your thought process in creating this page here?
> 
> ![image](https://user-images.githubusercontent.com/83927869/218222591-63d73101-5fd8-42b4-a525-74ebba7af7e5.png)

Yeah sure, I thought it would be helpful to provide a step by step guide, with visual aids on how to setup documentation for an existing project, with a focus on naming conventions and managing iterations visually.

**Comment by liz-zheng** [comment]
> ### 2/9/2023
> @liz-zheng Can you add any comments and thoughts about what changes you've made for this newest iteration?
> 
> ![image](https://user-images.githubusercontent.com/83927869/218222706-f7dfc5c2-149c-4ed2-8603-3a5430a47e20.png)

Changes: 
- removed step by step guide and created separate sections for documentation concerns with regards to the page layout and note content 
- Did some research into existing projects and made note of what can be improved and incorporated that into the designs
- Provide examples to show how each guideline can be implemented

**Comment by allthatyazz** [comment]
- During one of the PM CoP meetings, I asked other PMs how the designers on their team document their process. @liz-zheng please see the notes here https://github.com/hackforla/product-management/issues/131#issuecomment-1414849746. Some of the information might be helpful.

**Comment by liz-zheng** [comment]
Changes:

Added more bullet points for designer and developer notes section after interviewing Flora Osmond from Home Unite Us project about pain points 

![image](https://user-images.githubusercontent.com/15348137/222247156-59ca018e-026c-43bf-8670-647fa18ff778.png)
